locale/en/blog/vulnerability/mar-2022-security-releases.md

```diff
@@ -1,19 +1,40 @@
 ---
-date: 2022-03-16T23:22:00.000Z
+date: 2022-03-18T09:00:00.000Z
```

Comment:
I can't remember if we ran into issues if the date/time was in the future, so we might want to check this value before merging.

Reply:
Yup. Was just about to update it to be more accurate. Thanks!